core/src/main/java/ai/timefold/solver/core/api/score/stream/tri/TriConstraintStream.java

```diff
@@ -1892,10 +1915,10 @@ default Constraint penalizeBigDecimal(String constraintPackage, String constrain
      * For non-int {@link Score} types use {@link #penalizeConfigurableLong(String, ToLongTriFunction)} or
      * {@link #penalizeConfigurableBigDecimal(String, TriFunction)} instead.
      *
-     * @deprecated Prefer {@link #penalizeConfigurable(ToIntTriFunction)}.
      * @param constraintName never null, shows up in {@link ConstraintMatchTotal} during score justification
      * @param matchWeigher never null, the result of this function (matchWeight) is multiplied by the constraintWeight
      * @return never null
+     * @deprecated Prefer {@link #penalize(Score, ToIntTriFunction)} and {@link ConstraintWeightOverrides}.
      */
     @Deprecated(forRemoval = true)
```

Comment:
Should we include version `1.13.0` and update other stream classes?

Reply:
Not sure here. I left it here like this, because it was already deprecated before. Saying it's "since 1.13.0" wouldn't be true, because it was deprecated years ago already.